src/core/document.js

```diff
@@ -390,8 +390,9 @@ class Page {
       "XObject",
     ]);
 
+    const { xref } = this;
```

Comment:
The changes in this file looks unrelated, and can be reverted as far as I can tell.